cluster/images/etcd/migrate/versions.go

```diff
@@ -124,16 +124,6 @@ func ParseEtcdVersionPair(s string) (*EtcdVersionPair, error) {
 	return &EtcdVersionPair{version, storageVersion}, nil
 }
 
-// MustParseEtcdVersionPair parses a "<version>/<storage-version>" string to an EtcdVersionPair
-// or panics if the parse fails.
-func MustParseEtcdVersionPair(s string) *EtcdVersionPair {
```

Comment:
This function is used in integration test: 
https://github.com/kubernetes/kubernetes/blob/c7f9dd0bafe2f3328148a85eed9c18322b9f308e/cluster/images/etcd/migrate/integration_test.go#L71

Reply:
![image](https://user-images.githubusercontent.com/13084946/63562721-46c8b800-c591-11e9-97c1-ca5383ca25c6.png)

I am confused.What should i do?